vignettes/articles/data_from_epireview.Rmd

```diff
@@ -0,0 +1,270 @@
+
+```{r, match-parameter-to-article}
+article_row <- match(marburg_incub$id, marburg_articles$id)
+article_row
+```
+
+```{r, subset-articles}
+marburg_incub_article <- marburg_articles[article_row, ]
+marburg_incub_article
+```
+
+Now we can repeat the example of converting to `<epidist>` as shown above, but this time pass the bibliographic information as well as the epidemiological parameter information to create a full citation. The bibliographic information needs to be passed with the `articles` argument. 
+
+```{r, convert-to-epidist-full-citation}
+marburg_incub_epidist <- as_epidist(
+  marburg_incub,
+  articles = marburg_incub_article
+)
+marburg_incub_epidist
+```
+
+```{r, show-full-citation}
+marburg_incub_epidist$citation
+```
+
+::: {.alert .alert-info}
+The `as_epidist()` function is an S3 generic. If you are not familiar with S3 object-oriented programming in R, then this detail is not important, however, it does mean that the `articles` argument is not explicitly in the function definition of `as_epidist()` (i.e. it will not show up on autocomplete when typing out the function and will not be shown if you read the function help page `?as_epidist()`). Instead, the argument is specified as part of the `...` argument. This is because the `articles` argument is only required when converting data from {epireview} into `<epidist>`, and other data that can be converted to `<epidist>` objects do not require this argument.
+:::
+
+## Multi-row {epireview} entries
+
+The way the {epireview} data is stored means that some epidemiological parameter entries require multiple rows. This can be, for example, because they contain two summary statistics (e.g. mean and standard deviation) that are kept on separate rows. In order to create `<epidist>` objects that contains the full information for each entry multiple rows of the epidemiological parameters table from {epireview} can be given to `as_epidist()` to create a single `<epidist>` object.
+
+We can search in the data which entries have multiple rows by checking if there are duplicated parameter types and IDs. Remembering that we are using only the delay distributions (i.e. Human delay) from the epidemiological parameters, that we subset above. 
+
+```{r, check-multi-row-entries}
+multi_row_entries <- duplicated(marburg_params$parameter_type) &
+  duplicated(marburg_params$id)
+multi_row_ids <- marburg_params$id[multi_row_entries]
+```
+
+```{r, subset-multi-row-entries}
+multi_row_marburg_params <-
+  marburg_params[marburg_params$id %in% multi_row_ids, ]
+```
+
+This step should be verified manually to ensure that the entries that have been selected are indeed multiple rows for the same reported epidemiological parameter. We use the first two rows of this subset table, which are the mean and standard deviation for the generation time of Marburg disease. 
```

Comment:
I think there can definitely be a function in the future that checks that multiple rows correspond to the same entry. It would be better if this function were part of {epireview} as it relates to their data. 

Ideally, we would move away from multi-row entries if possible and I believe that the epireview team are trying to avoid multi-row entries for all future systematic reviews.

In terms of "so that the epidist object contains all relevant parameters", this is tricky as what parameters the `<epidist>` object stores is dependent on what data from {epireview} is supplied to `as_epidist()`, for now we are dependent on the entry having some usable data (e.g. mean and SD or distribution parameters).